src/script/vco-ie/main_warhammer/wh_dlc08_nor_norsca/missions.lua

```diff
@@ -89,14 +89,30 @@ local missions = {
 					region wh3_main_combi_region_the_awakening;
 					region wh3_main_combi_region_wei_jin;
 					region wh3_main_combi_region_zharr_naggrund;
+				} 
+
+				objective
+				{
+					override_text mission_text_text_vco_nor_wul_3_wheres_my_palico;

```

Comment:
Why is `vco_nor_wul_3_wheres_my_palico` here and not part of the victory `vco_victory_type_nor_wul_3_wheres_my_palico`? Same with the `vco_victory_payload_nor_wul_3_monster_hunt` bonus.

Reply:
I thought u wanted it moved to act 2 but leave it in act 3 if you want.

Reply:
I do not care where it is, but change the names if it is going to be in the 2nd.

Reply:
It's staying in route 3, this stuff should be picked out long before release.